server/build/babel/preset.js

```diff
@@ -1,5 +1,15 @@
-const babelRuntimePath = require.resolve('babel-runtime/package')
-  .replace(/[\\/]package\.json$/, '')
+// Used to generate paths for hard-coding module aliases that don't include the
+// path back to the root of the filesystem. Important for when building and
+// running are done from different directories.
+const packageRelativeResolve = (path) => {
+  // We want paths relative to the first 'node_modules' we find in the path
+  const RELATIVE_START = 'node_modules/'
+
+  const absolutePath = require.resolve(path)
+    .replace(/[\\/]package\.json$/, '')
+
+  return absolutePath.substring(absolutePath.indexOf(RELATIVE_START) + RELATIVE_START.length)
+}
```

Comment:
An early-out should fix the link/test issue:

```js
const packageRelativeResolve = (path) => {
  // We want paths relative to the first 'node_modules' we find in the path
  const RELATIVE_START = 'node_modules/'

  const absolutePath = require.resolve(path)
    .replace(/[\\/]package\.json$/, '')

  if (absolutePath.indexOf(RELATIVE_START) === -1) {
    // next.js is not being used as a package (either it is linked or we’re running tests) so it doesn’t matter if we use absolute paths or not
    return absolutePath
  }

  return absolutePath.substring(absolutePath.indexOf(RELATIVE_START) + RELATIVE_START.length)
}
```

Otherwise you end up with half-baked paths if there's no `node_modules` (for example, I ran this branch locally and got):

```
var _css = require('mes/Documents/Development/[...]/next.js/dist/lib/css.js');
```